pandas_gbq/gbq.py

```diff
@@ -6,57 +6,19 @@
 import time
 import sys
 
+import google.auth
```

Comment:
But I included google-auth in the dependencies in setup.py? What case would the try imports protect against?

gbq is an optional dependency, but once you install it, pip will install all of its dependencies.